pkg/agent/proxy/proxier_test.go

```diff
@@ -78,6 +80,8 @@ var (
 
 	skippedServiceNN = "kube-system/kube-dns"
 	skippedClusterIP = "192.168.1.2"
+
+	loadBalancerIPModeProxy = corev1.LoadBalancerIPModeProxy
```

Comment:
you can use `ptr.To(corev1.LoadBalancerIPModeProxy)` to get a pointer of a constant.

Reply:
Thanks, will update.